google-cloud-spanner-cassandra/src/main/java/com/google/cloud/spanner/adapter/Launcher.java

```diff
@@ -72,114 +119,384 @@ public class Launcher {
   private static final String PORT_PROP_KEY = "port";
```

Comment:
should we consolidate all configuration keys and default values into a single `Constants` or `ConfigDefaults` class? wdyt?